cdk/domino_cdk/config/eks.py

```diff
@@ -1,8 +1,11 @@
 from dataclasses import dataclass
-from typing import Dict, List
+from typing import Dict, List, Optional
+from logging import Logger
 
 from domino_cdk.config.util import MachineImage, check_leavins, from_loader
 
+log = Logger("domino_cdk.config")
```

Comment:
We don't actually use this and its corresponding import (was for an earlier version of the PR).